tokio/Cargo.toml

```diff
@@ -58,14 +66,9 @@ process = [
 ]
 # Includes basic task execution capabilities
 rt-core = []
-# TODO: rename this -> `rt-threaded`
-rt-full = [
-  "macros",
+rt-threaded = [
```

Comment:
why `threaded` vs `thread-pool`? I find threaded to be a bit odd?

Reply:
The two runtime variants were renamed to “basic_scheduler” and “threaded_scheduler” in a previous commit. This brings the feature flag in line with that name. 

We can open an issue to revisit that naming if you wish.